vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -198,6 +198,15 @@ import hudson.AbortException
      * use this volume in an initContainer.
      */
     'containerMountPath',
+     /**
+     * as `dockerImage` for the init container
```

Comment:
This is not a complete sentence. Please add a description that is helpful to the user reading our documentation.